README.md

```diff
@@ -152,7 +152,7 @@ running by other teams.
 ## Future Work
 
 - Fine-grained authorization and auditing
-- Secure Istio components (mixer, discovery service, etc.)
+- Secure Istio components (Mixer, Istio-Manager, etc.)
```

Comment:
I'm following the conventions here: https://istio.io/docs/concepts/what-is-istio/overview.html. There it has Mixer, Istio-Manager